test-data/cloudfront/getDistribution-NotExists.json

```diff
@@ -0,0 +1,26 @@
+{
+  "message": "Missing credentials in config",
```

Comment:
This isn't an error to indicate that the distribution does not exist - it's an authentication issue.